File: packages/screen-reader/ios/Plugin/Classes/Events/StateChangeEvent.swift
import Foundation
import Capacitor

@objc public class StateChangeEvent: NSObject, Result {
    let enabled: Bool

    init(enabled: Bool) {
        self.enabled = enabled
    }

    @objc public func toJSObject() -> AnyObject {
        var result = JSObject()
        result["enabled"] = enabled
        return result as AnyObject
    }
}

File: packages/screen-reader/android/src/main/java/io/capawesome/capacitorjs/plugins/screenreader/classes/events/StateChangeEvent.java
package io.capawesome.capacitorjs.plugins.screenreader.classes.events;

import androidx.annotation.NonNull;
import com.getcapacitor.JSObject;
import io.capawesome.capacitorjs.plugins.screenreader.interfaces.Result;

public class StateChangeEvent implements Result {

    private final boolean enabled;

    public StateChangeEvent(boolean enabled) {
        this.enabled = enabled;
    }

    @Override
    @NonNull
    public JSObject toJSObject() {
        JSObject result = new JSObject();
        result.put("enabled", enabled);
        return result;
    }
}
